code/src/Components/Album.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+const Album = (props) => {
+  const ArtistList = props.artists.map((artist) => (
+    <span>{artist.name}</span>
+  ))
+
+  let output = [] 
+  ArtistList.forEach((artist, index) => {
+    output.push(artist)
+    if (index !== ArtistList.length - 1) {
+      output.push(', ')
+    }
+    console.log(output)
+  })
+ 
+  return (
+    <div class="album">
+      <div class="image">
+        <img src={props.img} alt="Album"></img>
```

Comment:
For the `alt` here maybe you can use the title or something to describe what the image is?

Reply:
good point, updated!